app/schema.py

```diff
@@ -18,16 +20,20 @@ class Mentor(BaseModel):
     industry_knowledge: bool
     pair_programming: bool
     commitment: bool
-    referred_by: constr(max_length=255)
+    referred_by: Optional[constr(max_length=255)]
     other_info: Optional[constr(max_length=2500)]
-    validate_status: constr(max_length=255)
+    validate_status: Literal['approved', 'rejected', 'pending']
+    is_active: bool
+    accepting_new_mentees: bool
 
     class Config:
         extra = Extra.forbid
 
 
 class MentorUpdate(BaseModel):
-    profile_id: Optional[constr(max_length=255)]
+    profile_id: constr(max_length=255)
+    created_at: Optional[datetime]
```

Comment:
created_at should not be an alterable field - it is set when the mentor is created. Please remove created_at from MentorUpdate collection.